getting-started/README.md

```diff
@@ -73,51 +74,73 @@ brew install python3.7
 python3.7 -m pip install --upgrade setuptools pip virtualenv
 ```
 
-#### Linux (Ubuntu)
+## Ubuntu (native or WSL)
 
 First add the *deadsnake* ppa repos which contain Python3.7 binaries:
 
 ```
-sudo apt update
-sudo apt install software-properties-common
+sudo apt-get update
+sudo apt-get install software-properties-common
 sudo add-apt-repository ppa:deadsnakes/ppa
-sudo apt update
+sudo apt-get update
 ```
 Now, install Python 3.7 along with pip and virtualenv:
 ```
-sudo apt install python3.7
-sudo apt install python3-pip
+sudo apt-get install python3.7
+sudo apt-get install python3-pip
```

Comment:
@gdcutting split the code block here for terminal output and message saying to not worry about warnings